### PR TITLE
WS-xx-fix-usercontroller-update

### PR DESCRIPTION
**Pull request by Purbai**
changed httppost to httpput in update endpoint in usercontroller
tested using swagger to update user 6 and checked in PGAdmin that user was updated successfully

### CODE REVIEW

File: backend/Controllers/UserController.cs
Comment (RuksanEmirali) [inline backend/Controllers/UserController.cs:1]: This is fine, thanks for fixing our typo.

File: frontend/src/App.tsx
Comment (RuksanEmirali) [inline frontend/src/App.tsx:1]: Should this be in this ticket? Looks like it might be for something else.

File: frontend/src/pages/Login/Login.tsx
Comment (RuksanEmirali) [inline frontend/src/pages/Login/Login.tsx:1]: Should this be in this ticket? Looks like it might be for something else.

**Review by RuksanEmirali — COMMENTED** [review]
Well spotted on this typo. Some clarification is required for 2 of your changes, looks like they might have been for a different ticket.

File: frontend/src/pages/Login/Login.tsx
Comment (sun-amy) [reply frontend/src/pages/Login/Login.tsx:1]: It has to be done to pass the test on commit

**Review by Kiran-R-K — APPROVED** [review]
well done

**Pull request merged**